go.mod

```diff
@@ -11,19 +11,19 @@ replace (
 
 require (
 	github.com/blang/semver/v4 v4.0.0
-	github.com/cilium/charts v0.0.0-20230526205117-9c16d4d680bb
+	github.com/cilium/charts v0.0.0-20230601164911-0a270e4bf15f
 	github.com/cilium/cilium v1.14.0-snapshot.2
 	github.com/cilium/hubble v0.11.5
 	github.com/cilium/proxy v0.0.0-20230420184820-797bea843de1
-	github.com/cilium/tetragon/pkg/k8s v0.0.0-20230530162345-26256999192b
+	github.com/cilium/tetragon/pkg/k8s v0.0.0-20230601145017-408202913e4c
 	github.com/cilium/workerpool v1.2.0
 	github.com/cloudflare/cfssl v1.6.4
 	github.com/go-openapi/strfmt v0.21.7
 	github.com/google/gops v0.3.27
 	github.com/mholt/archiver/v3 v3.5.1
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 	github.com/spf13/cobra v1.7.0
-	github.com/spf13/pflag v1.0.6-0.20210604193023-d5e0c0615ace
+	github.com/spf13/pflag v1.0.6-0.20200504143853-81378bbcd8a1
```

Comment:
I'm not sure why this is downgraded. Looks like we need something like https://github.com/cilium/hubble/blob/cb12f7f274938d82705b81486af05ffb908595f8/.github/renovate.json5#L111-L114 in cilium-cli as well.

Reply:
https://github.com/cilium/cilium-cli/pull/1691